tensorboard/uploader/exporter_test.py

```diff
@@ -248,6 +264,179 @@ def stream_experiment_data(request, **kwargs):
             },
         )
 
+    def test_e2e_success_case_with_only_tensors_data(self):
+        mock_api_client = self._create_mock_api_client()
+        mock_api_client.StreamExperiments.return_value = iter(
+            [_make_experiments_response(["789"])]
+        )
+
+        def stream_experiments(request, **kwargs):
+            del request  # unused
+            self.assertEqual(kwargs["metadata"], grpc_util.version_metadata())
+
+            response = export_service_pb2.StreamExperimentsResponse()
+            response.experiments.add(experiment_id="123")
```

Comment:
why is response experiment_id "123" here but "789" at line 270?

Reply:
Line 270 is actually superfluous, because `mock_api_client.StreamExperiments` is overridden below (Line 305). Removed Line 270.